### ISSUE TITLE
[BUG] Error when updating a model that has a list

### ISSUE THREAD

**Issue by robsonrk** [laraform/laraform#21]
### Prerequisites

* [X] Able to reproduce the behaviour outside of your code, the problem is isolated to Laraform.
* [X] Provided all the required dependencies to reproduce the bug (eg. form, component, model, migration).
* [X] Your issue isn't already filed.
* [X] You filled in the entire issue template.

### Versions

* PHP version: 7.3.11
* Laravel version: 6.6.0
* Laraform Vue package version: 1.1.8
* Laraform Laravel package version: 1.1.7

### Description
Can't update a model with relation through Laraform with list of objects.

#### Laravel's User migration:
```
Schema::create('users', function (Blueprint $table) {
    $table->bigIncrements('id');
    $table->string('name');
    $table->string('email')->unique();
    $table->timestamp('email_verified_at')->nullable();
    $table->string('password');
    $table->rememberToken();
    $table->timestamps();
});
```

#### User model
```
// app/User.php

// relation to Contact model
public function contacts()
{
    return $this->hasMany(Contact::class);
}
```

#### Contacts model migration:
```
Schema::create('contacts', function (Blueprint $table) {
    $table->bigIncrements('id');

    $table->unsignedBigInteger('user_id');
    $table->string('type')->nullable();
    $table->string('label')->nullable();
    $table->string('value')->nullable();

    $table->foreign('user_id')
        ->references('id')
        ->on('users')
        ->onUpdate('cascade')
        ->onDelete('cascade');

    $table->timestamps();
});
```

#### Form
```
<?php

namespace App\Forms;

use Laraform;
use App\User;

class IssueForm extends Laraform
{
    public $model = User::class;

    public $buttons = [
        [
            'label' => 'Save',
            'class' => 'btn-info',
        ]
    ];

    public $schema = [
        'id' => [
            'type' => 'key'
        ],
        'name' => [
            'type' => 'text',
            'label' => 'Name',
            'rules' => 'required'
        ],
        'email' => [
            'type' => 'text',
            'label' => 'Email',
            'rules' => 'required'
        ],
        'password' => [
            'type' => 'password',
            'label' => 'Password',
            'rules' => 'required'
        ],
        'contacts' => [
            'type' => 'list',
            'label' => 'Contacts',
            'object' => [
                'schema' => [
                    'type' => [
                        'type' => 'select',
                        'label' => 'Type',
                        'placeholder' => '-- select one --',
                        'items' => [
                            'phone' => 'Phone',
                            'email' => 'Email'
                        ],
                        'rules' => 'required'
                    ],
                    'label' => [
                        'type' => 'text',
                        'label' => 'Label',
                        'description' => 'ex.: Friends phone',
                        'rules' => 'required'
                    ],
                    'value' => [
                        'type' => 'text',
                        'label' => 'Phone',
                        'rules' => 'required'
                    ]
                ]
            ]
        ]
    ];

    public function after()
    {
        response()->json([
            "status" => "success",
            "messages" => ["Saved!"]
        ])->send();
    }
}
```

#### Route
```
Route::get('/form', function () {
    $form = app('App\Forms\IssueForm');

    $user = App\User::with('contacts')->where('id',4)->first();
    $form->setData($user->toArray());
    
    return view('form',[
        'form' => $form,
    ]);
});
```

#### View
```
// resources/views/for.blade.php

@extends('layouts.app')

@section('content')
    <div class="container">
        <h1>ok</h1>
        <div class="row justify-content-center">
            <div class="col-md-6">
                <div class="card card-body">
                    {!! $form->render() !!}
                </div>
            </div>
        </div>
    </div>
@endsection
```

**Expected behavior:**

It's expected to update the model along with the relation.

**Actual behavior:** 
When updating a model with a list, it trows the following error:
```
POST http://localhost:8000/laraform/process 500 (Internal Server Error)

exception: "UnexpectedValueException"
file: "/var/www/vendor/laraform/laraform-laravel/src/Elements/ListElement.php"
line: 138
message: "Attribute value must be an array, object given"
```

### Additional Information
Modifying the source code solved the problem:
```
// src/Elements/ListElement.php
// line: 134
--  $originalCollection = $entity->{$this->attribute} ?? [];
++  $originalCollection = json_decode($entity->{$this->attribute},true) ?? [];
```

**Comment by laraform** [comment]
Hi @robsonrk, thank you for the detailed issue! I've looked into it and released a backend patch (1.1.8) that solves the problem, so please update. Btw, your solution was pretty close!

**Comment by robsonrk** [comment]
Thank you very much for the fast resolution.
